File: PSP/Proyectos/ServidorRest_DavidRoldan/src/main/java/domain/modelo/Customer.java
package domain.modelo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Customer {
    private int id;
    private String firstname;
    private String lastname;
    private String email;
    private String phone;
    private LocalDate dob;
}

File: Móviles/Proyectos/Login_DavidRoldan/app/src/main/java/com/example/login_davidroldan/domain/modelo/Customer.kt
package com.example.login_davidroldan.domain.modelo

import java.time.LocalDate

data class Customer (
    val id: Int,
    val firstname: String,
    val lastname: String,
    val email: String?,
    val phone: String?,
    val dob: LocalDate?,
    var isSelected: Boolean = false,
)